crowdstrike_cre/lib/falconpy/_endpoint/_falconx_sandbox.py

```diff
@@ -0,0 +1,387 @@
+        "required": True
+      }
+    ]
+  ],
+  [
+    "QueryReports",
+    "GET",
+    "/falconx/queries/reports/v1",
+    "Find sandbox reports by providing an FQL filter and paging details. "
+    "Returns a set of report IDs that match your criteria.",
+    "falconx_sandbox",
+    [
+      {
+        "type": "string",
+        "description": "Optional filter and sort criteria in the form of an FQL query. "
+        "For more information about FQL queries, see [our FQL documentation in Falcon]"
+        "(https://falcon.crowdstrike.com/support/documentation/45/falcon-query-language-feature-guide).",
+        "name": "filter",
+        "in": "query"
+      },
+      {
+        "type": "string",
+        "description": "The offset to start retrieving reports from.",
+        "name": "offset",
+        "in": "query"
+      },
+      {
+        "type": "integer",
+        "description": "Maximum number of report IDs to return. Max: 5000.",
+        "name": "limit",
+        "in": "query"
+      },
+      {
+        "type": "string",
+        "description": "Sort order: `asc` or `desc`.",
+        "name": "sort",
+        "in": "query"
+      }
+    ]
+  ],
+  [
+    "QuerySubmissions",
+    "GET",
+    "/falconx/queries/submissions/v1",
+    "Find submission IDs for uploaded files by providing an FQL filter and paging details. "
+    "Returns a set of submission IDs that match your criteria.",
+    "falconx_sandbox",
+    [
+      {
+        "type": "string",
+        "description": "Optional filter and sort criteria in the form of an FQL query. "
+        "For more information about FQL queries, see [our FQL documentation in Falcon]"
+        "(https://falcon.crowdstrike.com/support/documentation/45/falcon-query-language-feature-guide).",
+        "name": "filter",
+        "in": "query"
+      },
+      {
+        "type": "string",
+        "description": "The offset to start retrieving submissions from.",
+        "name": "offset",
+        "in": "query"
+      },
+      {
+        "type": "integer",
+        "description": "Maximum number of submission IDs to return. Max: 5000.",
+        "name": "limit",
+        "in": "query"
+      },
+      {
+        "type": "string",
+        "description": "Sort order: `asc` or `desc`.",
+        "name": "sort",
+        "in": "query"
+      }
+    ]
+  ],
+  [
+    "GetSampleV2",
+    "GET",
+    "/samples/entities/samples/v2",
+    "Retrieves the file associated with the given ID (SHA256)",
+    "falconx_sandbox",
+    [
+      {
+        "type": "string",
+        "description": "User UUID",
+        "name": "X-CS-USERUUID",
+        "in": "header"
+      },
+      {
+        "type": "string",
+        "description": "The file SHA256.",
+        "name": "ids",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "string",
+        "default": False,
+        "description": "Flag whether the sample should be zipped and password protected with pass='infected'",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 5b9a0a8c-c8fc-490d-a228-3914065c87ad -->